packages/components/Menu/src/MenuItemCheckbox/useMenuItemCheckbox.ts

```diff
@@ -120,9 +122,22 @@ export const useMenuCheckboxInteraction = (
     hasTooltips,
   };
 
+  const onFocusOverride = React.useCallback(
+    (e) => {
+      pressable.props.onFocus(e);
+      // if it's not a hover focus
+      if (!pressable.state.focused && !pressable.state.hovered) {
```

Comment:
Is `hover focus` a windows term? I assume it means "mouse hover". I feel like I'd find this to be more clear if it was worded like "If not focused via mouse hover"